frontend/assets/top_containers.bulk.js

```diff
@@ -203,8 +203,74 @@ BulkActionIlsHoldingUpdate.prototype.show = function() {
     selection: this.bulkContainerSearch.get_selection()
   });
 
+  var $modal = AS.openCustomModal("bulkUpdateModal", this.$menuItem[0].text, dialog_content, 'full');
 
-  var $modal = AS.openCustomModal("bulkUpdateModal", this.$menuItem.text(), dialog_content, 'full');
+  this.setup_update_form($modal);
+};
+
+
+/***************************************************************************
+ * BulkActionContainerProfileUpdate - Container Profile bulk action
+ *
+ */
+function BulkActionContainerProfileUpdate(bulkContainerSearch) {
+  this.bulkContainerSearch = bulkContainerSearch;
+  this.MENU_ID = "bulkActionUpdateContainerProfile";
+
+  this.setup_menu_item();
+};
+
+
+BulkActionContainerProfileUpdate.prototype.setup_update_form = function($modal) {
+  var self = this;
+
+  var $form = $modal.find("form");
+
+  $(document).trigger("loadedrecordform.aspace", [$form]);
```

Comment:
Hey @jambun.  I've removed the <script> from the template and replaced it with this "loadedrecordform.aspace" event trigger.  This will ensure any widgets/form behaviours in the template are initialised for us.